### PR TITLE
SGP number in unrelease page

### PR DESCRIPTION
**Pull request by seenanair**
Mandatory SGP Number field added in Unrelease page

### CODE REVIEW

File: src/pages/Unrelease.tsx [hunk 1]
```diff
@@ -26,7 +27,8 @@ const validationSchema = Yup.object().shape({
         barcode: Yup.string().required(),
         highestSection: Yup.number().min(0, 'Section number must be greater than or equal to 0').notRequired()
       })
-    )
+    ),
+  workNumber: Yup.string().required('SGP Number is a required field')
```
Comment (khelwood) [inline src/pages/Unrelease.tsx:31]: Is it mandatory? The story does not specify that.

Comment (seenanair) [reply src/pages/Unrelease.tsx:31]: Ohh, is it? I thought it is mandatory. I will ask

Comment (seenanair) [reply src/pages/Unrelease.tsx:31]: @khelwood  Katy has confirmed that the SGP number is mandatory

**Review by khelwood — APPROVED** [review]

**Comment by seenanair** [comment]
@khelwood Please review the changes to associate work number with all labware scanned for unrelease operation

[https://github.com/sanger/stan-client/pull/257/commits/a16234df2d399733fbbc30ea612df2172b89cfe9](https://github.com/sanger/stan-client/pull/257/commits/a16234df2d399733fbbc30ea612df2172b89cfe9)

**Review by khelwood — APPROVED** [review]

**Pull request merged**